en/orm/associations.rst

```diff
@@ -356,11 +356,11 @@ convention:
 ========================== ===================
 Relation                   Schema
 ========================== ===================
-Article hasMany Comment    Comment.article\_id
+Articles hasMany Comments  Comments.article\_id
 -------------------------- -------------------
```

Comment:
You might have to extend the table underlines here. Sphinx is particular about lines matching their contents.

Reply:
Always miss something with those.